code/datums/materials/basemats.dm

```diff
@@ -142,6 +142,17 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	qdel(source.GetComponent(/datum/component/slippery))
 	qdel(source.GetComponent(/datum/component/squeak))
 
+///Quiet and/or hard to see.
+/datum/material/tranquillite
+	name = "tranquillite"
+	id = "tranquillite"
+	desc = "Material with quiet and clear properties"
+	color = "#ffffff" //white
```

Comment:
why is it visible this is cringe

Reply:
Okay admittedly while making literally everything make from tranquillite would be a bit amusing, i think it might be a bit overboard.

It's already going to have (real, permanent, gas-blocking) invisible walls, invisible floors, etc... making even the sheet invisible might be overkill.

Though, hmm... maybe if it does the fadeout thing like that one genetic mutation.

Reply:
make it have like 10 alpha or somehting